docs/content/documentation/content/shortcodes.md

```diff
@@ -134,6 +134,24 @@ If you want to have some content that looks like a shortcode but not have Zola t
 you will need to escape it by using `{%/*` and `*/%}` instead of `{%` and `%}`. You won't need to escape
 anything else until the closing tag.
 
+### Invocation Count
+
+Every shortcode context is passed in a variable named `invocation` that tracks how many times a particular shortcode has
```

Comment:
Ok the code looks good but I'm not sold on the name `invocation`. `nth`? `nth_invocation`? `invocation_count`?

Reply:
I agree that the name leaves something to be desired. I think `nth` or `invocation_count` are probably the best options; `invocation_count` is more descriptive, but in my subjective opinion it feels too verbose relative to the rest of Zola's features/API. How much concern is there around breaking users' existing shortcodes if we were to go with `nth`?

Reply:
> How much concern is there around breaking users' existing shortcodes if we were to go with nth?

None I think?